### PR TITLE
Probability enhancement

### PR DESCRIPTION
**Pull request by EmilieNumworks**

### CODE REVIEW

File: escher/include/escher/table_cell.h [hunk 1]
```diff
@@ -19,9 +19,9 @@ class TableCell : public HighlightCell {
   View * subviewAtIndex(int index) override;
   void layoutSubviews() override;
   constexpr static KDCoordinate k_separatorThickness = 1;
-private:
   constexpr static KDCoordinate k_labelMargin = 10;
   constexpr static KDCoordinate k_accessoryMargin = 10;
```
Comment (EmilieNumworks) [inline escher/include/escher/table_cell.h]: Should it simplify the table cell layout?

File: apps/probability/law/law.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ class Law : public Shared::CurveViewRange {
   virtual double evaluateAtDiscreteAbscissa(int k) const;
 protected:
   constexpr static int k_maxNumberOfOperations = 1000000;
-  constexpr static double k_maxProbability = 0.9995;
+  constexpr static double k_maxProbability = 0.9999995;
```
Comment (EmilieNumworks) [inline apps/probability/law/law.h]: Infer from LargeNumberOfSignificative....

File: apps/probability/calculation_controller.h [hunk 3]
```diff
@@ -32,19 +33,21 @@ class CalculationController : public ViewController, public Shared::ParameterTex
...
     LawCurveView * lawCurveView();
     ImageTableView * imageTableView();
     EditableTextCell * calculationCellAtIndex(int index);
     void willDisplayEditableCellAtIndex(int index);
     constexpr static int k_maxNumberOfEditableFields = 3;
   private:
-    constexpr static KDCoordinate k_largeTextFieldWidth = 60;
-    constexpr static KDCoordinate k_textFieldWidth = 50;
+    constexpr static KDCoordinate k_minTextFieldWidth = 4*KDText::charSize().width();
+    constexpr static KDCoordinate k_maxTextFieldWidth = 10*KDText::charSize().width();
```
Comment (EmilieNumworks) [inline apps/probability/calculation_controller.h]: Add the margin in order to avoid weird blinking

**Comment by AlainBusser** [comment]
Thanks alot for this new year's  :gift: 
The 2 last changes of pull request #215 are useless now, since I had reprogrammed erf function. It remains only the first two points which are

1. The 1.96 confidence interval
2. The words "analyse" and "calculus" instead of "calculation" for whatever concerns the derivative etc...

**Pull request merged**